blazor/diagram/accessibility.md

```diff
@@ -0,0 +1,86 @@
+<kbd>Ctrl+Shift+Left Angle Bracket (<)</kbd> | This command is used to decrease the font size of the selected text.
+<kbd>Ctrl+Shift+L</kbd> | This command is used to align the selected text to the left.
+<kbd>Ctrl+Shift+C</kbd> | This command is used to center the selected text horizontally.
+<kbd>Ctrl+Shift+R</kbd> | This command is used to align the selected text to the right.
+<kbd>Ctrl+Shift+J</kbd> | This command is used to justify the selected text, aligning it to both the left and right margins.
+<kbd>Ctrl+Shift+E</kbd> | This command is used to align the selected text to the top vertically. 
+<kbd>Ctrl+Shift+M</kbd> | This command is used to center the selected text vertically.
+<kbd>Ctrl+Shift+V</kbd> | This command is used to align the selected text to the bottom vertically.
+<kbd>Ctrl+G</kbd> or <kbd>Ctrl+Shift+G</kbd> | This command is used to group together multiple selected shapes, allowing them to be treated as a single shape.
+<kbd>Ctrl+Shift+U</kbd> | This command is used to ungroup shapes within a previously grouped selection.
+<kbd>Ctrl+Shift+F</kbd> | This command is used to bring the selected shape forward in the stacking order, making it appear in front of other shapes.
+<kbd>Ctrl+Shift+B</kbd> | This command is used to send the selected shape backward in the stacking order, making it appear behind other shapes.
+<kbd>Ctrl+]</kbd> | Visually moves the selected node, connector, and group over the nearest overlapping node, connector, or group.
+<kbd>Ctrl+[</kbd> | Visually moves the selected node, connector, and group behind the underlying node or connector or Group.
+<kbd>Ctrl+L</kbd> | This command is used to rotate the selected nodes counterclockwise.
+<kbd>Ctrl+R</kbd> | This command is used to rotate the selected nodes clockwise.
+<kbd>Ctrl+H</kbd> | This command is used to flip the selected diagram elements horizontally.
+<kbd>Ctrl+J</kbd> | This command is used to flip the selected diagram elements vertically.
+<kbd>Tab</kbd> | Select the diagram elements in Forward: Focus on the diagram element based on the rendering order when using the "Tab" key.
+<kbd>Shift + Tab</kbd> | Select the diagram elements in backward: Focus on the diagram element based on the rendering order when using the "Shift+Tab" key.
+<kbd>Ctrl+D</kbd> | Duplicate the selected shape.
+<kbd>Enter</kbd> | Perform annotation editing for the selected diagram element.
+<kbd>Ctrl+1</kbd> | This command is used to activate the pointer tool.
+<kbd>Ctrl+3</kbd> | This command is used to activate the connector tool.
+<kbd>Ctrl+2</kbd> | This command is used to activate the text tool.
+<kbd>Ctrl+8</kbd> | This command is used to activate the rectangle tool.
+<kbd>Ctrl+6</kbd> | This command is used to activate the line tool.
+<kbd>Ctrl+5</kbd> | This command is used to activate the freeform tool.
+<kbd>Ctrl+9</kbd> | This command is used to activate the ellipse tool.
+
```

Comment:
| Interaction Keys | Description |
| ---------------- | ----------- |
| <kbd>Ctrl+C</kbd> | Copy the selected diagram elements to the clipboard. |
| <kbd>Ctrl+X</kbd> | Cut the selected diagram elements to the clipboard. |
| <kbd>Ctrl+V</kbd> | Paste the diagram elements from the clipboard. |
| <kbd>Ctrl+A</kbd> | Select all the diagram elements. |
| <kbd>Delete</kbd> | Delete the selected diagram elements. |
| <kbd>Ctrl+P</kbd> | Print the diagram page. |
| <kbd>Ctrl+Z</kbd> | Undo the last action. |
| <kbd>Ctrl+Y</kbd> | Redo the last undo action. |
| <kbd>ArrowUp</kbd> | Move the selected diagram elements upwards by the specified delta value. |
| <kbd>ArrowDown</kbd> | Move the selected diagram elements downwards by the specified delta value. |
| <kbd>ArrowRight</kbd> | Move the selected diagram elements to the right by the specified delta value. |
| <kbd>ArrowLeft</kbd> | Move the selected diagram elements to the left by the specified delta value. |
| <kbd>F2</kbd> | Start editing the selected diagram elements. |
| <kbd>Ctrl++</kbd> | Zoom in the diagram. |
| <kbd>Ctrl+-</kbd> | Zoom out the diagram. |
| <kbd>Ctrl+B</kbd> | Toggle bold formatting for the selected text. |
| <kbd>Ctrl+I</kbd> | Toggle italic formatting for the selected text. |
| <kbd>Ctrl+U</kbd> | Toggle underline formatting for the selected text. |
| <kbd>Ctrl+Shift+Right Angle Bracket (&gt;)</kbd> | Increase the font size of the selected text. |
| <kbd>Ctrl+Shift+Left Angle Bracket (&lt;)</kbd> | Decrease the font size of the selected text. |
| <kbd>Ctrl+Shift+L</kbd> | Align the selected text to the left. |
| <kbd>Ctrl+Shift+C</kbd> | Center the selected text horizontally. |
| <kbd>Ctrl+Shift+R</kbd> | Align the selected text to the right. |
| <kbd>Ctrl+Shift+J</kbd> | Justify the selected text, aligning it to both the left and right margins. |
| <kbd>Ctrl+Shift+E</kbd> | Align the selected text to the top vertically. |
| <kbd>Ctrl+Shift+M</kbd> | Center the selected text vertically. |
| <kbd>Ctrl+Shift+V</kbd> | Align the selected text to the bottom vertically. |
| <kbd>Ctrl+G</kbd> or <kbd>Ctrl+Shift+G</kbd> | Group selected shapes together, treating them as a single shape. |
| <kbd>Ctrl+Shift+U</kbd> | Ungroup shapes within a previously grouped selection. |
| <kbd>Ctrl+Shift+F</kbd> | Bring the selected shape forward in the stacking order. |
| <kbd>Ctrl+Shift+B</kbd> | Send the selected shape backward in the stacking order. |
| <kbd>Ctrl+]</kbd> | Move the selected node, connector, and group over the nearest overlapping node, connector, or group. |
| <kbd>Ctrl+[</kbd> | Move the selected node, connector, and group behind the underlying node, connector, or group. |
| <kbd>Ctrl+L</kbd> | Rotate the selected nodes counterclockwise. |
| <kbd>Ctrl+R</kbd> | Rotate the selected nodes clockwise. |
| <kbd>Ctrl+H</kbd> | Flip the selected diagram elements horizontally. |
| <kbd>Ctrl+J</kbd> | Flip the selected diagram elements vertically. |
| <kbd>Tab</kbd> | Select the diagram elements forward based on the rendering order. |
| <kbd>Shift + Tab</kbd> | Select the diagram elements backward based on the rendering order. |
| <kbd>Ctrl+D</kbd> | Duplicate the selected shape. |
| <kbd>Enter</kbd> | Perform annotation editing for the selected diagram element. |
| <kbd>Ctrl+1</kbd> | Activate the pointer tool. |
| <kbd>Ctrl+3</kbd> | Activate the connector tool. |
| <kbd>Ctrl+2</kbd> | Activate the text tool. |
| <kbd>Ctrl+8</kbd> | Activate the rectangle tool. |
| <kbd>Ctrl+6</kbd> | Activate the line tool. |
| <kbd>Ctrl+5</kbd> | Activate the freeform tool. |
| <kbd>Ctrl+9</kbd> | Activate the ellipse tool. |